# awesome-genome
[![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)

### *Machine Learning in Genomics*

- [deep_learning_DNA](https://github.com/onceupon/deep_learning_DNA) - Classifying human DNA sequence and random ATCG sequences, using keras CNN
- [DeepMod](https://github.com/WGLab/DeepMod) - takes long-read signals as input and outputs modification summary for each genomic position in a reference genome together with modification prediction for each base in a long read. 
- [janggu](https://github.com/BIMSBbioinfo/janggu) - a python package that facilitates deep learning in the context of genomics
- [genomelake](https://github.com/kundajelab/genomelake) 
- [deepvariant](https://github.com/google/deepvariant) - deep learning-based variant caller
- [Keras-genomics](https://github.com/gifford-lab/Keras-genomics) - A Keras-based deep learning platform to perform hyper-parameter tuning, training and prediction on genomics data

### related
- https://github.com/danforthcenter/plantcv

### gene expression
- **2016-02** | Gene expression inference with deep learning | *Yifei Chen, Yi Li, Rajiv Narayan, Aravind Subramanian, Xiaohui Xie* | [Bioinformatics](https://doi.org/10.1093/bioinformatics/btw074)


### regulatory region
- **2015-03** | Deep Feature Selection: Theory and Application to Identify Enhancers and Promoters | *Yifeng Li, Chih-Yu Chen, and Wyeth W. Wasserman* | [19th Annual International Conference, RECOMB 2015, Warsaw, Proceedings](https://doi.org/10.1007/978-3-319-16706-0_20)

- **2015-07** | DeepBind: Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning | *Babak Alipanahi,	 Andrew Delong,	Matthew T. Weirauch & Brendan J. Frey* | [Nature Biotechnology](https://doi.org/10.1038/nbt.3300)

- **2015-08** | Deep learning for regulatory genomics | *Yongjin Park & Manolis Kellis* | [Nature Biotechnology](https://doi.org/10.1038/nbt.3313)

- **2015-08** | DeepSEA: Predicting effects of noncoding variants with deep learningâ€“based sequence model | *Jian Zhou & Olga G. Troyanskaya* | [Nature Methods: Short intro](https://doi.org/10.1038/nmeth.3604) & [Nature Methods](https://doi.org/10.1038/nmeth.3547)

- **2016-03** | Genome-Wide Prediction of cis-Regulatory Regions Using Supervised Deep Learning Methods | *Yifeng Li, Wenqiang Shi, Wyeth W Wasserman* | [Biorxiv](https://doi.org/10.1101/041616)



### other 
- https://github.com/gokceneraslan/awesome-deepbio
- https://github.com/josephmisiti/awesome-machine-learning
- https://github.com/visenger/awesome-mlops
- https://github.com/windmaple/awesome-AutoML
- https://github.com/danielecook/Awesome-Bioinformatics#readme


### License

[![CC0](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](https://creativecommons.org/publicdomain/zero/1.0/)


